Library/Homebrew/cmd/update-report.rb

```diff
@@ -586,70 +542,152 @@ def add(reporter, auto_update: false)
   def dump(updated_formula_report: true)
     # Key Legend: Added (A), Copied (C), Deleted (D), Modified (M), Renamed (R)
 
-    dump_formula_report :A, "New Formulae"
-    if updated_formula_report
-      dump_formula_report :M, "Updated Formulae"
+    unless Homebrew::EnvConfig.update_report_all_formulae?
+      dump_formula_or_cask_report :A, "New Formulae"
+      dump_formula_or_cask_report :AC, "New Casks"
+      dump_formula_or_cask_report :R, "Renamed Formulae"
```

Comment:
I feel like formula renames might be relevant if the user has them installed (though the dump will need updating to actually check that).

Reply:
If they are installed they were automatically migrated anywhere and the user shouldn't really need to care.

Reply:
Yeah once trying it out locally it made more sense now.